packages/local_auth/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -40,6 +42,13 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
 
 #pragma mark Private Methods
 
+- (LAContext *)authContext {
+  if (_authContextOverride != nil) {
+    return _authContextOverride;
+  }
+  return [[LAContext alloc] init];
```

Comment:
Each `self.authContext` is going to call the constructor (when no  _authContextOverride is provided)

If we always wants a new instance (which i think that's the case), we should rename it do something doesn't sound too much like a getter`. Maybe:
`- (LAContext *)getNewAuthContext`?

Reply:
Renamed to `createAuthContext` since `create` is a standard idiom. Also future-proofed the override by making it better fit the fact that this is really, as you point out, a factory rather than a getter; it now uses an array of overrides, so a test could run a flow with multiple API calls.